OutSystems/outsystems-datagrid-reactive
# ROU-3052: Sort client actions should return error message and success boolean

This PR adds feedback messages to Sort API methods. Both ClearSort and ColumnSort methods did not return any feedback on whether or not the operation was successful or if any errors happened.

### Test Steps
1. Click in "Column Sort By Product Name":
Expected: Grid should be sorted according to the Product Name column (ascending) and a success message should appear. 

1. Click in "Clear  Sort Function":
Expected: Grid should lose the sorting by Product Name column and a success message should appear. 

1. Click in "Column Sort By Invalid Column":
Expected: Error message should show saying: "Error"

1. Click in "Clear  Sort Invalid":
Expected: Error message should show saying: "Error"



### Checklist
